tools/src/main/scala/scala/scalanative/build/NativeConfig.scala

```diff
@@ -251,8 +251,10 @@ object NativeConfig {
     def withCompileOptions(value: Seq[String]): NativeConfig =
       copy(compileOptions = value)
 
-    def withTargetTriple(value: Option[String]): NativeConfig =
+    def withTargetTriple(value: Option[String]): NativeConfig = {
+      System.setProperty("target.triple", value.getOrElse(""))
```

Comment:
Let's set this system property only when it is defined instead of setting empty string. Even better lets clear it when changing target triple Some(???) -> None, this way we would not get into troubles when modifying config
```suggestion
      val propertyName = "target.triple"
      value match { 
         case Some(triple) => System.setProperty(propertyName, triple)
         case None =>         System.clearProperty(propertyName)
       }
```

Reply:
makes sense.
I will run tests and hope to be able to push these changes before too long ...

Reply:
done